src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -710,11 +704,6 @@ static void close_transport_locked(grpc_chttp2_transport* t,
             t->keepalive_ping_timer_handle.reset();
           }
         }
-        if (t->keepalive_watchdog_timer_handle.has_value()) {
```

Comment:
why is this being removed?

Reply:
it's subsumed by the timeout per-outstanding-ping